src/wiki/plugins/links/mdx/djangowikilinks.py

```diff
@@ -57,7 +57,7 @@ def extendMarkdown(self, md, md_globals):
         self.md = md
 
         # append to end of inline patterns
-        WIKI_RE = r'\[(?P<linkTitle>[^\]]+?)\]\(wiki:(?P<wikiTitle>[a-zA-Z0-9\./_-]*?)\)'
+        WIKI_RE = r'\[(?P<linkTitle>[^\]]+?)\]\(wiki:(?P<wikiTitle>[a-zA-Z0-9\./_#-]*?)\)'
```

Comment:
It would be nicer to use a named group for the fragment, otherwise this is allowed: `wiki:slug#blah#blah`

    r'\[(?P<linkTitle>[^\]]+?)\]\(wiki:(?P<wikiTitle>[a-zA-Z0-9\./_-]*?)(?P<anchor>#[a-zA-Z0-9\./_-]*?)\)'